.github/workflows/ci.yml

```diff
@@ -91,6 +91,17 @@ jobs:
         command: check
         args: --all
 
+    - name: Check devcontainer matches MSRV
+      run: |
+        versions=$(sed -nE 's|^FROM (.*/)?rust:([^ ]+)|\2|p' .devcontainer/Dockerfile)
+        echo "Versions: $versions"
+        mismatches=$(echo "$versions" | grep -v '^${{ steps.msrv.outputs.msrv }}' || true)
+        echo "Mismatches: $mismatches"
+        if [ -n "$mismatches" ]; then
+          echo "::error ::Devcontainer includes incorrect rust version(s): $mismatches"
+          exit 1
+        fi
+
```

Comment:
I confirmed that this fails when the devcontainer does not match the MSRV

Reply:
that's great - thanks a lot!